make_image.sh

```diff
@@ -0,0 +1,231 @@
+#!/bin/bash
+
+set -o errtrace
+set -o nounset
+
+ROOT="$( cd $( dirname $0 ); pwd )"
+
+TOOLCHAIN_DIR_DEFAULT="/opt/grisp/grisp2-rtems-toolchain/rtems/5"
```

Comment:
There is no canonical default path to the toolchain, so hardcoding this here is not needed.

Reply:
It is the default for me, so at least I don't need to pass the -t argument XD

Reply:
But that’s not portable to anyone else’s environment. Please remove it. I guess you could export the environment variable, then you don’t have to pass the flag either?

Reply:
I removed it